### PR TITLE
Genetics portal as new source of evidence

### PR DESCRIPTION
**Pull request by DSuveges**
GWAS Catalog source is removed. Replaced with describing the OT Genetics portal as new source of GWAS based evidences.

### CODE REVIEW

File: data-sources/genetic-associations.md [hunk 1]
```diff
@@ -4,9 +4,12 @@ Genetic associations are available for both common and rare \(Mendelian\) diseas
...
+
+From February 2020 genome-wide association based evidences are sourced from the [Open Targets Genetics portal](https://genetics.opentargets.org/). Genome Wide Association Studies (GWAS) provide a link between common diseases and genomic loci, where each association is represented by a lead variant where the evidence for the association is the strongest. As the lead variant is not necessarely causal (or the only causal) variant of the associated locus, and the causal gene(s) are not neccesarely the closest to the lead variant, it is challenging to identify target/disease associations based on GWAS data. The genetics portal provides pipelines to identify tagging variants for the lead variant through LD expansion or credible set analysis, and a machine learning based method to link the associated locus to causal genes based on the summarized effect of the tagging variants. For more information on the applied method, see Genetics portal [documentation](https://genetics-docs.opentargets.org/our-approach/pipeline-overview). The weight of an evidence is given by the locus to gene score describing the likelihood of the gene being causal for the associated locus.
```
Comment (d0choa) [inline data-sources/genetic-associations.md]: Replace "machine learning based method to link the associated locus to causal genes based on the summarised effect of the tagging variants."
with 
"machine learning method (L2G) to link the associated locus to the most likely causal gene(s) by integrating the summarized effect of the tagging variants using multiple genetic and functional genomic features (e.g. eQTL, pQTL, pCHI-C, etc.)."

File: data-sources/genetic-associations.md [hunk 2]
```diff
@@ -4,9 +4,12 @@ Genetic associations are available for both common and rare \(Mendelian\) diseas
...
+
+The genetics portal import summary statistics and manually curated lead variants from the NHGRI-EBI [GWAS Catalog](https://www.ebi.ac.uk/gwas) and UK Biobank summary statistics from the [Neale lab](http://www.nealelab.is/uk-biobank). For the target/disease evidence, GWAS associations with p-value below the genome-wide significant threshold (5\*10^-8) are considered.
```
Comment (d0choa) [inline data-sources/genetic-associations.md]: Replace the last sentence "For the target..." with:
In order to generate evidence connecting targets with diseases, significant GWAS associations (p-value < 5\*10^-8) are assigned to the most likely causal gene(s) using the L2G machine learning method. The score provided by the method is used for evidence scoring (link to section?).

Comment (DSuveges) [reply data-sources/genetic-associations.md]: Requested changes applied in [02cb4a05e6a6a1c1a0f043c06900b1d46342249b](https://github.com/opentargets/docs.targetvalidation.org/pull/11/commits/02cb4a05e6a6a1c1a0f043c06900b1d46342249b)

File: data-sources/genetic-associations.md (hunk 1 above)
Comment (DSuveges) [reply data-sources/genetic-associations.md]: Requested changes applied in [02cb4a05e6a6a1c1a0f043c06900b1d46342249b](https://github.com/opentargets/docs.targetvalidation.org/pull/11/commits/02cb4a05e6a6a1c1a0f043c06900b1d46342249b)

**Pull request merged**